composer/trainer/trainer.py

```diff
@@ -1436,8 +1437,8 @@ def fit(
 
         # Grad Accum
         if grad_accum is not None:
-            self.adaptive_gradient_accumulation = _is_adaptive_grad_accum(grad_accum, device=self._device)
-            if self.adaptive_gradient_accumulation and self.state.profiler:
+            self.state.auto_grad_accum = _is_adaptive_grad_accum(grad_accum, device=self._device)
```

Comment:
since you are renaming here, also rename `_is_adaptive_grad_accum` to `_is_auto_grad_accum`?